admin/controllers/pupil.js

```diff
@@ -95,27 +82,142 @@ const postAddPupil = async (req, res, next) => {
   } catch (error) {
     next(error)
   }
-  res.redirect(`/school/pupil-register/lastName/true?hl=${pupil._id}`)
+  const pupilId = JSON.stringify([pupil._id])
+  res.redirect(`/school/pupil-register/lastName/true?hl=${pupilId}`)
```

Comment:
As a suggestion, I would make this into a friendly URL (no ?=)

Reply:
Setting the highlight via the query string in my work.   I would rather use the qs than session, personally.

Reply:
@pris54 we need to pass down a list of people that will display the green indicator.

I am more than happy to hear your suggestions on how to achieve this.